Add scroll detection to OverlayPanelContent and DialogContent (#4294)
jpmorganchase/salt-ds: .changeset/silent-ties-sing.md

---
"@salt-ds/core": patch
---

> Added right padding in `OverlayPanelContent` between content and scrollbar when overflow

Reviewer: ```suggestion
Added padding between the content and scrollbar when `OverlayPanelContent` is scrollable.
```
?
